Runtimes/Supplemental/StringProcessing/cmake/caches/Vendors/Apple/apple-common.cmake

```
@@ -0,0 +1,9 @@
+set(CMAKE_BUILD_TYPE RelWithDebInfo CACHE STRING "")
+set(BUILD_SHARED_LIBS YES CACHE BOOL "")
```

We would need to build these with whole module optimization to ensure we don't regress the library size.

```suggestion
set(BUILD_SHARED_LIBS YES CACHE BOOL "")
set(CMAKE_Swift_COMPILATION_MODE wholemodule CACHE String "")
```

---

Good point, sorry for missing that.